src/app/retrospective/retrospective.module.ts

```diff
@@ -5,27 +5,34 @@ import { NgModule } from '@angular/core';
 import { CustomMaterialModule } from '../core/custom-material/custom-material.module';
 import { SharedModule } from '../shared/shared.module';
 import { RetrospectiveRoutingModule } from './retrospective-routing.module';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 // Import Components
 import { RetrospectiveListComponent } from './retrospective-list/retrospective-list.component';
 import { RetrospectiveCreateComponent } from './retrospective-create/retrospective-create.component';
-import { ReactiveFormsModule } from '@angular/forms';
+import { SprintDetailComponent } from './sprint-detail/sprint-detail.component';
+import { BasicModalComponent } from '../shared/basic-modal/basic-modal.component';
 
 
 @NgModule({
     imports: [
         CommonModule,
-        SharedModule,
+        FormsModule,
         ReactiveFormsModule,
         CustomMaterialModule,
+        SharedModule,
         RetrospectiveRoutingModule
     ],
     declarations: [
         RetrospectiveListComponent,
         RetrospectiveCreateComponent,
+        SprintDetailComponent
     ],
     exports: [],
-    providers: []
+    providers: [],
+    entryComponents: [
+        BasicModalComponent
```

Comment:
**RetrospectiveCreateComponent** should be listed in the **entryComponents** instead of **declarations**, *since we are using retrospective create component as a modal and it should be loaded when the application is bootstrapped*

Reply:
Even it should be declared in the shared module itself, not here